### PR TITLE
Language: Russian language update

### PR DESCRIPTION
**Pull request by scientistnt**
I simply translated almost all strings which weren't translated.

### CODE REVIEW

**Comment by saibotk** [comment]
Thank you for your contribution :rocket: 

@Satton2 could you give this translation a look, to get a second opinion, as none of us sadly does understand Russian.

**Comment by Satton2** [comment]
> Thank you for your contribution 
> 
> @Satton2 could you give this translation a look, to get a second opinion, as none of us sadly does understand Russian.
> 
> PS: Make sure to add a line to the `CHANGELOG.md` file ;)

Sure. Will do that within 1-2 days, okay? Btw, could you assign me to this PR if possible?

**Comment by saibotk** [comment]
@Satton2 Thank you, take your time. And i assigned you, just needed you to comment, otherwise i cannot assign you :+1:

**Comment by Satton2** [comment]
@scientistnt 
Should I wait a day or more so you can complete editing your translation? Maybe you forgot something else? ;)

When you will be sure that your translation is done, let me know, and I will find some time to review it.

**Comment by scientistnt** [comment]
> @scientistnt
> Should I wait a day or more so you can complete editing your translation? Maybe you forgot something else? ;)
> 
> When you will be sure that your translation is done, let me know, and I will find some time to review it.

I'm never sure about anything, but the localization seems to be finished and ready for review (i really hope i won't need to change anything again soon).

**Comment by Satton2** [comment]
@scientistnt 
> I'm never sure about anything, but the localization seems to be finished and ready for review (i really hope i won't need to change anything again soon).

It's sad that "you never sure about anything". Anyway, you will definitely need to change your translation after my review ;)

Btw, have you synchronized it with my (official) translation of TTT? Make sure that all non-modified strings of TTT are translated in the same way in TTT2.

Also, I hurriedly review your translation and saw that you translated "addon" as "аддон" which is wrong and must be changed to "дополнение". Make sure to synchronize your translation with the official Russian translation of the game.

**Review by Satton2 — CHANGES_REQUESTED** [review]
Finally, I got some time to make this review.

@scientistnt Please, change all these strings when you will be able, and this pull request will be ready for the merge.

**Note**: If you have some questions about the requested changes, feel free to ask.

File: gamemodes/terrortown/gamemode/shared/lang/ru.lua [hunk 1]
```diff
@@ -1040,122 +1040,122 @@ L.corpse_binoculars = "[{key}]: осмотреть тело через Бино
...
+L.submenu_guide_roles_title = "Роли"
+L.submenu_guide_equipment_title = "Снаряжение"
+
+L.submenu_bindings_bindings_title = "Клавиши"
+
+L.submenu_language_language_title = "Язык"
+
+L.submenu_appearance_general_title = "Общее"
+L.submenu_appearance_hudswitcher_title = "Сменить интерфейс"
 --L.submenu_appearance_vskin_title = "VSkin"
```
[the remark this answers is not in the document]
Comment (Satton2) [reply gamemodes/terrortown/gamemode/shared/lang/ru.lua:1111]:
```suggestion
L.submenu_appearance_vskin_title = "Скин интерфейса"
```

File: gamemodes/terrortown/gamemode/shared/lang/ru.lua [hunk 2]
```diff
@@ -1164,132 +1164,132 @@ L.label_crosshair_thickness = "Толщина прицела"
...
+L.item_no_drown_damage = "Иммунитет к урону от утопления"
+L.item_no_drown_damage_desc = [[Делает вас неуязвимым к урону от утопления.]]
 
 -- 2020-04-21
---L.dna_tid_possible = "Scan possible"
---L.dna_tid_impossible = "No scan possible"
---L.dna_screen_ready = "No DNA"
---L.dna_screen_match = "Match"
+L.dna_tid_possible = "Сканирование возможно"
+L.dna_tid_impossible = "Сканирование невозможно"
```
Comment (Satton2) [inline gamemodes/terrortown/gamemode/shared/lang/ru.lua:1249]:
```suggestion
L.dna_tid_possible = "Сканирование возможно."
L.dna_tid_impossible = "Сканирование невозможно."
```

**Comment by Satton2** [comment]
TTT2 team, this PR currently relies on changes in the following PR request: https://github.com/TTT-2/TTT2/pull/710
Review that PR first.

**Review by Satton2 — APPROVED** [review]
Since https://github.com/TTT-2/TTT2/pull/710 is now merged, I approve these changes.
Thank you for your work!

Please update the PR to resolve the conflicts. Also, please move the changelog line to the "Changed" section.

**Review by saibotk — APPROVED** [review]

**Review by Histalek — APPROVED** [review]

**Pull request merged**